README.md

```diff
@@ -5,7 +5,7 @@ plugin.
 
 ## Roadmap
 
-This plugin is currently in Alpha and looking for contributors. To begin with it will aim to support the upload 
```

Comment:
What's the deal with these changes? Are the necessary in this PR? I am of the opinion they are not.

Reply:
Agreed. This is Resharper suggestions kicking in for me here. I'll revert the unnecessary changes here